go.mod

```diff
@@ -162,6 +162,8 @@ require (
 	sigs.k8s.io/yaml v1.3.0 // indirect
 )
 
+require github.com/HdrHistogram/hdrhistogram-go v1.1.2
+
```

Comment:
I wonder why it's in the separate section?

Reply:
good question, moving